### PR TITLE
PE-86/Resume sync after switching focus

### PR DESCRIPTION
**Pull request by jdaev**
Use document.visibilityState to detect if tab is hidden instead of document.hidden.

### CODE REVIEW

**Comment by jdaev** [comment]
> 
> 
> Do you need this in production as well?

Seems so. Don't know why, but if I use document.hidden it stops the sync when I switch back to the tab. Stops the stream entirely. It's strange because there isn't any error thrown either.

**Review by fedellen — CHANGES_REQUESTED** [review]
See suggested change above, change 'tab hidded' to 'tab is hidden'

File: lib/blocs/sync/sync_cubit.dart [hunk 1]
```diff
@@ -46,18 +46,18 @@ class SyncCubit extends Cubit<SyncState> {
...
-    print('syncing...');
-    if (await _profileCubit.logoutIfWalletMismatch()) {
-      emit(SyncWalletMismatch());
-      return;
-    }
-    emit(SyncInProgress());
-
     try {
+      if (window.document.visibilityState != 'visible') {
+        print('tab hidded, skipping sync...');
```
Comment (agsuy) [inline lib/blocs/sync/sync_cubit.dart:51]: typo here

**Review by jdaev — COMMENTED** [review]
Fixed the typo

**Review by fedellen — APPROVED** [review]
Looks good!

**Review by arielmelendez — APPROVED** [review]
:+1:

**Pull request merged**